### PR TITLE
Alexander Rybak HW2 Task3

### PR DESCRIPTION
**Pull request by Alexander-1313**

### CODE REVIEW

**Review by maximkliashchou — APPROVED** [review]
lgfm

**Review by barilo-egor — APPROVED** [review]
lgfm

**Review by asraysun — APPROVED** [review]
деление с остатком?

**Review by Tane4kka — APPROVED** [review]

File: students/alexander_rybak/homeWork2/task3/src/java/by/rybak/MathOperationsDemo.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        int firstNumber = 12;
+        int secondNumber = 4;
+
+        System.out.println("the first number is " + firstNumber);
+        System.out.println("the second number is " + secondNumber);
+
+        System.out.println("Addition(+)  = " + (firstNumber + secondNumber));
+        System.out.println("Subtraction(-) = " + (firstNumber - secondNumber));
+        System.out.println("Division(/) = " + (firstNumber / secondNumber));
+        System.out.println("Multiplication(*) = " + (firstNumber * secondNumber));
```
Comment (javaGuruBY) [inline students/alexander_rybak/homeWork2/task3/src/java/by/rybak/MathOperationsDemo.java:15]: You'd better avoid performing operations in sout

**Review by javaGuruBY — APPROVED** [review]

**Pull request merged**